### ISSUE TITLE
PPC Landing Pages

### ISSUE THREAD

**Issue by Katiemoser** [zesty-io/nextjs-website#452]
Please remove navigation bar from all /ppc/ landing pages. Only keep Zesty logo.

**Comment by darwin808** [comment]
![image](https://user-images.githubusercontent.com/71545960/163836356-061a6bdd-5b7a-4b13-859a-1b9a17d6c3b5.png)
@Katiemoser this header ?

**Comment by Katiemoser** [comment]
Yes, the live pages are showing the navigation from the main site.
